src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-weight: 500;
+    line-height: 14px;
+    color: #000;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__item:first-child {
```

Comment:
When an element is a flex or grid container, it's better to use the `gap` property instead of `margin`. This way, the spacing will only be applied between the items, and there will be no need to reset margins.

More about `gap`: https://www.w3schools.com/cssref/css3_pr_gap.php